force-clean.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
```

Comment:
useful file but not called or exposed via npm script - did you intend to include it?

Reply:
Wasn't intentional, but I could wrap it in an npm script, since it's useful after a fabric rebinding. So revert or npm script should it be?

Reply:
+1 for script

Reply:
Added as npm script